toolchains/local_java_repository.bzl

```diff
@@ -93,7 +93,7 @@ def local_java_runtime(name, java_home, version, runtime_name = None, visibility
     native.toolchain(
         name = "runtime_toolchain_definition",
         target_settings = [":%s_settings_alias" % name],
-        toolchain_type = "@bazel_tools//tools/jdk:runtime_toolchain_type",
+        toolchain_type = Label("@bazel_tools//tools/jdk:runtime_toolchain_type"),
         toolchain = runtime_name,
```

Comment:
I guess we don't need to add Label down in the line 230, becuase that's just a regular target in a BUILD file.